events/janus_rabbitmqevh.c

```diff
@@ -267,6 +277,59 @@ int janus_rabbitmqevh_init(const char *config_path) {
 	}
 
 	/* Connect */
+	int result=janus_rabbitmqevh_connect();
+	if(result < 0){
```

Comment:
A couple of code style things to fix:

1. please insert spaces where the equal sign is;
2. please insert a space between the closing curve bracket and the opening curly bracket too.

Reply:
Style updated